### ISSUE TITLE
Add expand UI based on the sunburst concept

### ISSUE THREAD

**Issue by jiemakel** [humanitiesplusdesign/fibra#26]

**Comment by esjewett** [comment]
There is an initial implementation in the master branch that I just deployed. It replaces the "Items" popover with a sunburst display for the moment.

Still to do
- [ ] Should the sunburst overlay the nodes or vice-versa?
- [ ] Color scheme for the sunburst

**Comment by jiemakel** [comment]
(There seems to be something wrong with the deployed dist version. I think I know what it is [I probably broke something when trying to fix dist version minification earlier], but it will take me until tomorrow to get things working again)

**Comment by esjewett** [comment]
I didn't even check - sorry. I suspect that ui-bootstrap isn't getting packaged. I'll look at it and see if I can work it out.

**Comment by jiemakel** [comment]
I'm pretty sure I broke it earlier, and the problem lies deep in the dependency injection part of the build magic. I just didn't run into the exact config that triggers this. I'll fix it.

**Comment by esjewett** [comment]
Yeah, not the angular-boostrap stuff, if I'm reading correctly. I will leave it to you. Thank you!

(For everyone else - `gulp serve` should work fine. I think.)
